crane/container.go

```diff
@@ -2,13 +2,14 @@ package crane
 
 import (
 	"fmt"
-	"github.com/flynn/go-shlex"
-	"github.com/michaelsauter/crane/print"
 	"io"
 	"os"
 	"path"
 	"strconv"
 	"strings"
+
+	"github.com/flynn/go-shlex"
+	"github.com/michaelsauter/crane/print"
```

Comment:
Did you change the order? The order of the imported packages is determined by `gofmt` and should not be changed. Or has `gofmt` output changed and I'm unaware of that?